src/app/components/Image/index.tsx

```diff
@@ -32,6 +32,7 @@ type Props = {
   src: string;
   width?: number;
   fetchpriority?: 'high';
+  overrideAspectRatio?: boolean;
```

Comment:
Another approach to this might be to simply apply these values if the aspect ratio is not supplied rather than providing an additional prop to apply them?

Also going back to your initial approach, if this is a really niche use case maybe it _was_ better to not modify the image component to cater for an edge case? Seeing what it looks like in code pushes me more in that direction as I can't really imagine a future maintainer making use of `overrideAspectRatio` given how specific its behaviour is.

Reply:
Both fair suggestions. I was a little heistant to change the Image component since a core feature of it seems to be that it maintains a set image ratio (16/9 being a fallback if no values are provided). 

I agree this is an edge case currently, but it's likely this logic will utilised by the Billboard component and Impactful Promo (which are new components currently with UX). So it appears it might be "less" of an edge case in the near future.

Reply:
Thanks for the consideration @Isabella-Mitchell, given the forecast requirement for this elsewhere, I'd reiterate my suggestion around applying this behaviour where no aspect ratio is provided. It looks to me like what you have defined is a set of sensible default styles where an aspect ratio is not provided; you can explicity provide the 16/9 fallback in places where the default is relied on currently. This moves away a confusing api where you could potentially provide the aspect ratio then configure to 'override' it with no clear indication as to what overriding actually does.

I had another look at the image component and saw some other bit of configuration we don't want to fragement with seperate impementations:
- [lazy loading configuration](https://github.com/bbc/simorgh/blob/102dc1d6266ec7c1d322bb2c924f9aedc8bd7415/src/app/components/Image/index.tsx#L165)
- [fetch priority](https://github.com/bbc/simorgh/blob/102dc1d6266ec7c1d322bb2c924f9aedc8bd7415/src/app/components/Image/index.tsx#L169)

If we make an image performance improvement it's good for it to work across the codebase.

Reply:
Cool, I'm developing this implementation and I think it makes sense.

Now I'm debating whether it's better to apply varying height styles using conditional (ternary) operator or overriding a default value.

E.g.

Conditional: [styles.wrapper, hasFixedAspectRatio ? styles.doThis : styles.elseDoThis]

Overriding: [styles.wrapper, hasFixedAspectRatio && styles.doThis]

I think overriding the overriding styles are easier to read, but that means nearly all Images will have two height declarations, which feels bloated.

Another approach is applying inline styles, but as Karina pointed out the combination of inline styles and CSS messy.